README.md

```diff
@@ -1,57 +1,18 @@
-# Northcoders News API
+# Welcome to Ali News 
 
-## Background
+## Initialising the databases
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+Now, you might be thinking Ali, this is a fantastic little news project you are building here, and you would be right. Now I know what you're thinking next, OMG there is no .env files and I cannot access your databases, don't worry little one I'm here to help.
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+First you will need to create two new files
+* .env.test
+* .env.development
 
-## Kanban
+Each of these will setup the database based on if you want the full database set or just some test data.\
+Now edit each of these files and copy the following text into each 'PGDATABASE=database_name_here'
 
-### Link to your Trello Board here: https://trello.com/b/7yiHe1nI
+for the .test file replace 'database_name_here' with 'nc_news_test'\
```

Comment:
You shouldn't be exposing your database names! A user can potentially use any database they like so long as they add an .env. file